compiler/codegen/OMRRegister.hpp

```diff
@@ -187,7 +184,6 @@ class OMR_EXTENSIBLE Register
       IsLive                        = 0x0010, // Register is currently live
       ContainsInternalPointer       = 0x0080,
       IsSinglePrecision             = 0x0400,
-      UpperHalfIsDead               = 0x0400, // GPRS ONLY -- no need to save/restore the high-order bits when spilled; aliased with IsSinglePrecision
```

Comment:
These two (`UpperHalfIsDead` and `IsSinglePrecision`) are aliased for some reason. I'm not sure if this is intentional or if this is mistake. Regardless I've checked that the `IsSinglePrecision` flag is only used for FPRs while the `UpperHalfIsDead` flag only affects GPRs so they shouldn't interfere.